myJoin.pl

```diff
@@ -18,68 +18,67 @@
 my @dup;
 
 #set input flags
-GetOptions ("H" => \$header,
-            "unique" => \$unique,
-            "duplicate" => \$duplicate,
-            "help" => \$help) or die ("Error in command line arguments -h for help\n");
+GetOptions("H" => \$header,
+"unique" => \$unique,
+"duplicate" => \$duplicate,
+"help" => \$help) or die("Error in command line arguments -h for help\n");
 
 #return help flag
-help_me() if ($help);
+help_me()  ($help);
 
-#check for incorrect usage of -d and -u
+#check for incorrect usage -d and -u
 die "Error in command line arguments --h for help\n" if ($duplicate && $unique);
 
 #command line input
 foreach(@ARGV){
-	line_hash1($_);
+    line_hash1($_);
 }
 
 #add hash keys to uniq and dup arrays
-if ($header){
+if($header){
```

Comment:
Diff makes it easy to catch these little spacing changes. for consistency, we should leave the space in.